python/CONTRIBUTING.md

```diff
@@ -71,3 +71,15 @@ For Python, we use pytest in combination with coverage to maintain 90% code cove
 ```bash
 make test
 ```
+
+To pass additional arguments to pytest, you can use `PYTEST_ARGS`.
```

Comment:
not sure if it would help, but one can discover list of potential arguments from https://docs.pytest.org/en/7.1.x/reference/reference.html#id89